mmf/trainers/core/training_loop.py

```diff
@@ -152,3 +143,16 @@ def _extract_loss(self, report: Dict[str, Any]) -> Tensor:
         loss_dict = report.losses
         loss = sum([loss.mean() for loss in loss_dict.values()])
         return loss
+
+    def _calculate_max_updates(self):
+        max_updates = self.training_config.max_updates
+        if max_updates is None:
+            max_updates = math.inf
+
+        max_epochs = self.training_config.max_epochs
+        if max_epochs is None:
+            epoch_max_updates = math.inf
+        else:
+            epoch_max_updates = len(self.train_loader) * max_epochs
+
```

Comment:
Let's add an assertion(with proper message) to check that user has specified atleast one of `max_updates` or `max_epochs`. If both are specified, we can take minimum as you are calculating. We can add a warning message specifying the priority and how it is calculated to the user.